### PR TITLE
[zh] sync stateful-apps.md in Chinese

### PR DESCRIPTION
**Pull request by windsonsea**
Signed-off-by: windsonsea <haifeng.yao@daocloud.io>

### Describe your changes
Added zh text to: 
```
content/zh-cn/stateful-apps.md
```

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-cn/stateful-apps.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: 有状态应用
+status: Completed
+category: 概念
+tags: ["基本原理", "", ""]
+---
+
+## 是什么
+
+当我们说到有状态（和[无状态](/zh-cn/stateless-apps/)）应用时，
```
Comment (Jacob953) [inline content/zh-cn/stateful-apps.md:10]: There is no `stateless-apps` under `/zh-cn`

**Review by Jacob953 — CHANGES_REQUESTED** [review]

File: content/zh-cn/stateful-apps.md (hunk 1 above)
Comment (Jacob953) [reply content/zh-cn/stateful-apps.md:10]: check it again, please. )

image

Comment (windsonsea) [reply content/zh-cn/stateful-apps.md:10]: Yes currently. PR #1472 has included the file of stateless-apps.

**Review by Jacob953 — APPROVED** [review]
Hold until #1472 is ready to merge.

**Comment by windsonsea** [comment]
> Hold until #1472 is ready to merge.

Could we move these two PRs forward if no more comments? Feel dirty to rebase while opening a new PR -_!

Another question is where and how I can apply a membership of this repo 😄 Thanks.

**Review by hanyuancheung — APPROVED** [review]

**Comment by hanyuancheung** [comment]
> > Hold until #1472 is ready to merge.
> 
> Could we move these two PRs forward if no more comments? Feel dirty to rebase while opening a new PR -_!
> 
> Another question is where and how I can apply a membership of this repo 😄 Thanks.

This PR lgtm. Feel free to keep contributing to this repo, and we elect memberships from time to time.

**Pull request merged**